substrate/frame/assets/src/lib.rs

```diff
@@ -1687,6 +1691,49 @@ pub mod pallet {
 			Self::deposit_event(Event::<T, I>::Blocked { asset_id: id, who });
 			Ok(())
 		}
+
+		/// Transfer the entire transferable balance from the caller asset account.
+		///
+		/// NOTE: This function only attempts to transfer _transferable_ balances. This means that
+		/// any held, frozen, or existential deposits (when `keep_alive` is `true`), will not be
+		/// transferred by this function. To ensure that this function results in a killed account,
+		/// you might need to prepare the account by removing any reference counters, storage
+		/// deposits, etc...
+		///
+		/// The dispatch origin of this call must be Signed.
+		///
+		/// - `id`: The identifier of the asset for the account holding a deposit.
+		/// - `dest`: The recipient of the transfer.
+		/// - `keep_alive`: A boolean to determine if the `transfer_all` operation should send all
+		///   of the funds the asset account has, causing the sender asset account to be killed
+		///   (false), or transfer everything except at least the existential deposit, which will
+		///   guarantee to keep the sender asset account alive (true).
```

Comment:
Assets have a `minimum_balance` but no existential deposit. This needs to be reworded. Also does `keep_alive` refer to `Accounts` in the context of the Assets pallet (i.e. `(AssetId, AccountId)`) or in the context of the system?

Reply:
In this case, it refers to the context of Assets pallet. I'll make sure to reword that part.